### ISSUE TITLE
Should all examples be built in the projectatomic namespace?

### ISSUE THREAD

**Issue by bexelbie** [projectatomic/nulecule-library#37]
Even if we put them in some other space, shouldn't all the examples be built and ready for a pull?

**Comment by dustymabe** [comment]
@bexelbie, yes. I'd like for them to be built and put into projectatomic's namespace.

That being said I would like to require that the images that the nulecule's depend on to be sourced either from the centos or fedora namespaces and not from our personal named repositories. 

There is also a discussion that could happen surrounding whether we are building a library that others can consume or not. I think some of our examples can be consumed by others (i.e. redis or mariadb), but others are complete applications that probably won't be integrated into part of another app. We should probably hold these two classes of apps to a separate standard.

**Comment by kadel** [comment]
> We should probably hold these two classes of apps to a separate standard.

+1

We should definitely start to separate what is just example and what is "library grade"

**Comment by bexelbie** [comment]
@dustymabe I agree.  It is important that we try to get examples using personal namespaces to not do so, even if we need to have additional docker files for the sub-components in this repo.  (Where they will be good examples of decomposition :) )
